firmware/application/app_settings.cpp

```diff
@@ -124,6 +124,7 @@ void SettingsStore::save() const {
 
 bool load_settings(std::string_view store_name, SettingBindings& bindings) {
     File f;
+    bool settings_loaded{false};
```

Comment:
I was thinking about this some more and this won't hurt to have, but it doesn't fix the underlying problem.
In apps like Scanner or Pocsag, there are additional settings (not radio settings) that will be loaded which will cause this to be "true".

What we really need is a way to tell if the radio settings are loaded and that logic doesn't belong here.
Maybe a better fix would be to keep the bindings for additional settings and radio settings separate in the SettingsManager. If we do that, we'll need to change the way settings are saved (because they will need to be saved piecewise).

Reply:
Actually, the more I think about it, the only nice way to handle this generically is probably to keep a flag on the binding indicating that the binding was set during loading.

Reply:
Maybe add a "was read" (not the best name) flag to the BoundSetting struct and set it in BoundSetting::parse if the parse was successful. Then in the SettingsManager we could test if the setting was actually read or not. That would keep the radio-specific code out of the generic settings code which would be good.

We could look at the "was read" for "VolumeLevel" because that is set by the radio settings for both RX and TX paths. That would probably be sufficient, but feels a little hacky.
Better would be to individually test each binding when copying settings to the radio model, but that's a lot more code and there really isn't a scenario where only some of the radio settings are read so a single test should be fine.